records/scrum_team_charter.md

```diff
@@ -0,0 +1,87 @@
+# Visual Meaning Scrum Team Charter
+
+## Things we always do
+
+- Try to write the best code
+
+- **Be kind**
+  - Assume best intent
+  - Bring humanity to work - full spectrum of emotion and acknowledgement that we are humans who do have personal lives
+  - Reach out and be vulnerable
+
+- **Commit**
+  - Do what we say we will do (and communicate when things change)
+  - Keep practical details front of mind
+  - Prioritise team outcomes
+  - Be meaningfully wrong
+
+- **Collaborate**
+  - Teach, share information and decisions, learning early and often and recording this where people can find the information again
+  - Broadcast by default
+
+- **Engage**
+  - Engage in synchronous sessions
+  - Thumbs up/down in response to questions in a group

```

Comment:
This sounds funny outside of the context. Can you just write "fast feedback and response to questions", please